hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsAsyncQueueHealth.kt

```diff
@@ -0,0 +1,101 @@
+package uk.gov.justice.hmpps.sqs
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.boot.actuate.health.Health
+import org.springframework.boot.actuate.health.Health.Builder
+import org.springframework.boot.actuate.health.HealthIndicator
+import software.amazon.awssdk.services.sqs.model.GetQueueAttributesRequest
+import software.amazon.awssdk.services.sqs.model.GetQueueAttributesResponse
+import software.amazon.awssdk.services.sqs.model.QueueAttributeName
+import kotlin.Result.Companion.failure
+import kotlin.Result.Companion.success
+
+class HmppsAsyncQueueHealth(private val hmppsQueue: HmppsAsyncQueue) : HealthIndicator {
+
```

Comment:
another candidate for shared code, I guess only the call to getAttributes is different?